keyboards/bastardkb/tbkmini/rules.mk

```diff
@@ -17,7 +17,10 @@ SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = yes            # USB Nkey Rollover
 BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE = yes     # Enable keyboard RGB matrix (do not use together with RGBLIGHT_ENABLE)
```

Comment:
`RGB_MATRIX_SUPPORTED` needs to be set to "yes", as well.

Reply:
Thanks for the catch.

Reply:
Welcome!  Otherwise, the "no" for the supported will forcibly deactivate the rgb matrix feature. :D